lib/DI/ContainerConfigurator.php

```diff
@@ -64,6 +64,8 @@ function configure(ContainerBuilder $container) {
     $container->autowire(\MailPoet\Config\Menu::class)->setPublic(true);
     $container->autowire(\MailPoet\Config\RendererFactory::class)->setPublic(true);
     $container->autowire(\MailPoet\Config\ServicesChecker::class);
+    $container->autowire(\MailPoet\Config\Shortcodes::class)
+      ->setShared(false);
```

Comment:
This `setShared` is not something we are used to. Don't you think we should add a comment here for others to understand what it does and why is it important to keep it here?

Reply:
Sure, I'll add an explanation.